apps/pie-docs/src/foundations/iconography/code.md

```diff
@@ -0,0 +1,277 @@
+```
+```sh
+yarn add @justeattakeaway/pie-icons-react
+```
+
+#### Import into your project
+Simply import the icons you need into your project:
+
+```tsx
+import { IconAlertTriangleLarge, IconCalendar } from "@justeattakeaway/pie-icons-react";
+
+export default function App() {
+  return (
+    <div className="App">
+      <IconCalendar />
+      <IconAlertTriangleLarge fill={PIE_ALIAS_COLOR_TOKEN} />
+    </div>
+  );
+}
+```
+
+{% notification {
+  type: "information",
+  message: "For more information, please refer to the package [README](https://github.com/justeattakeaway/pie/blob/main/packages/tools/pie-icons-react/README.md)."
+} %}
+
+---
+
+## Vue
+
+This package generates an icon set for Vue.js applications using the base [pie-icons](https://www.npmjs.com/package/@justeattakeaway/pie-icons) package. The SVGs in pie-icons are compiled into single file components that can be imported into Vue applications.
+
+{% notification {
+  type: "warning",
+  message: "The package is fully compatible with Vue 2. To use these icons in Vue 3, you may need to take some extra steps when building your application. Please reach out to **#help-designsystem** for assistance."
+} %}
+
+### Usage
+
+#### Installation
+You can install the package using `npm` or `yarn`:
+
+```sh
+npm install @justeattakeaway/pie-icons-vue --save
+```
+```sh
+yarn add @justeattakeaway/pie-icons-vue
+```
+
+#### Import into your project
+Simply import the icons you need into your project:
+
+```html
+  <template>
+      <icon-calendar />
+      <icon-alert-triangle-large fill={PIE_ALIAS_COLOR_TOKEN} />
+  </template>
+
+  <script>
+  import { IconCalendar, IconAlertTriangleLarge } from '@justeattakeaway/pie-icons-vue';
+
+  export default {
+      components: {
+          IconCalendar,
+          IconAlertTriangleLarge
+      }
+  };
+  </script>
+  ```
+
+{% notification {
+  type: "information",
+  message: "For more information, please refer to the package [README](https://github.com/justeattakeaway/pie/blob/main/packages/tools/pie-icons-vue/README.md)."
+} %}
+
+---
+## Web Components
+
+This package generates a framework-agnostic Web Component icon set for web applications. It uses the base [pie-icons](https://www.npmjs.com/package/@justeattakeaway/pie-icons) package. The SVGs in pie-icons are compiled into [Lit](https://lit.dev/) web components.
+
+{% notification {
+  type: "warning",
+  message: "Our Web Components package is still in **active development** and is not yet production ready. We will be adding integration guides for different frameworks in the near future."
+} %}
+
+### Usage - In Lit Components
+
+#### Installation
+
+```sh
+npm install @justeattakeaway/pie-icons-webc --save
+```
+```sh
+yarn add @justeattakeaway/pie-icons-webc
+```
+
+#### Import into your project
+We suggest importing the bundle for an individual component directly.
+
+```js
+import '@justeattakeaway/pie-icons-webc/icons/IconAppRestaurant';
```

Comment:
This will be different depending on if people are using TS or JS (weirdly, I just checked our modal and it looks like that points to the JS path even though it's TS!).

Not sure if it's worth mentioning that here though and the two different paths

```suggestion
import '@justeattakeaway/pie-icons-webc/icons/IconAppRestaurant'; // Import the typescript version of the icon
// OR
import '@justeattakeaway/pie-icons-webc/dist/icons/IconAppRestaurant.js'; // Import the ESM version of the icon
```

Reply:
Maybe not a discussion for this PR, but one thing I still need to wrap my head around is when people will want to import the TS version. If we provide a JS component, type declarations and a type map, do we still need to export the TS file?

Reply:
My honest answer is – I'm not sure. It's probably a question to ask the Skip engineers over what they'd expect from a component like this when using it in a TS application.

I'd be happy to just have the single export available if that makes it simpler to understand and doesn't detriment anything when importing the JS version in a TS app. If there's a clear benefit to using the TS version then would be interested in knowing it, as like you say, the JS version should provide everything they need anyway?